src/eko/couplings.py

```diff
@@ -1,6 +1,5 @@
-r"""Manage running (and non) couplings.
+r"""Manage running (and fixed) couplings.
 
-Contains the QCD beta function coefficients and the handling of the running
 coupling :math:`\alpha_s`.
```

Comment:
I guess @giacomomagni was more after the idea :upside_down_face: 
```suggestion
Manage QCD coupling :math:`\alpha_s` and QED coupling :math:`\alpha`.
We provide an interface to access them simultaneously and provide several
strategies to solve the associated |RGE|.
```